### ISSUE TITLE
Add more hiking related amenities to route layer

### ISSUE THREAD

**Issue by dittaeva** [waymarkedtrails/waymarked-trails-site#18]
I suggest the following should be added to all route layers, because they're bound to be of good use to the user:
amenity=bench
leisure=picnic_table
tourism=picnic_site
amenity=toilets

I am not sure if and how all of them are rendered on Mapnik, but at least the bench does not seem to be rendered, maybe the toilets are?

**Comment by qugebert** [comment]
I just discovered tagging for stamp locations, which could also be a potential candidate for this

https://wiki.openstreetmap.org/wiki/Key:checkpoint

**Comment by lonvia** [comment]
Yes, waymarkedtrails really should start to support checkpoints officially. I was hoping to sort out some of the underlying database code first before adding more amenities. So still might be a while....
